src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalService.kt

```diff
@@ -118,11 +127,15 @@ class CalculationTransactionalService(
     sourceData: PrisonApiSourceData,
     booking: Booking,
     userInput: CalculationUserInputs?,
+    approvedDates: List<ManualEntrySelectedDate>?,
   ): CalculatedReleaseDates {
     try {
       val calculation =
         calculate(booking, CONFIRMED, sourceData, userInput, calculationFragments)
-      writeToNomisAndPublishEvent(prisonerId, booking, calculation)
+      if (approvedDates != null) {
```

Comment:
just wondering if there is any benefit in splitting out the 'approved dates storage' into a separate endpoint? 

Although looks like it uses the calculation so makes sense to leave it here